### PR TITLE
Using ByteStringer where appropriate and removing places that are not.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ScanAdapter.java [hunk 1]
```diff
@@ -96,8 +97,8 @@ public Builder adapt(Scan scan, ReadHooks readHooks) {
         .setFilter(filter)
         .setRowRange(
             RowRange.newBuilder()
-                .setStartKey(ByteStringer.wrap(scan.getStartRow()))
-                .setEndKey(ByteStringer.wrap(scan.getStopRow())));
+                .setStartKey(ByteString.copyFrom(scan.getStartRow()))
+                .setEndKey(ByteString.copyFrom((scan.getStopRow()))));
```
Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ScanAdapter.java]: ByteString.copyFrom(scan.getStopRow ?

**Comment by kevinsi4508** [comment]
What is the difference between ByteStringer.wrap() and ByteString.copyFrom() again?

**Comment by sduskis** [comment]
ByteStringer.wrap() uses the original byte[] and ByteString.copyFrom() makes a new copy.

**Comment by kevinsi4508** [comment]
How do we know when to use ByteStringer.wrap() and when to use ByteString.copyFrom()? It is not obvious to me by looking at those codes.

**Comment by sduskis** [comment]
It's safer to use  ByteString.copyFrom(), but it comes at a performance cost.  We should only use  ByteStringer.wrap() when we can guarantee that the underlying byte[] will not be manipulated while we're using it in an RPC.

**Comment by kevinsi4508** [comment]
What you said makes sense. let me take a look and see if I can figure out byte[] is not going to be changed at places where we use ByteStringer.wrap().

**Comment by kevinsi4508** [comment]
The places that we use ByteStringer.wrap() seem to be wrapping local byte[]s.
LGTM

**Pull request merged**